### ISSUE TITLE
Dev a11y swarm for psammead bulleted list

### ISSUE THREAD

**Issue by pharingee** [BBC-archive/psammead#3761]
We need to a11y swarm the psammead bulleted list component to ensure it meets accessibility criteria. Things to be considered during this swarm are:

 - all our supported screen readers and that everything is as expected
-  I should be able to see the bullet point when I change my colour preferences to a black background in FireFox
- I should be able to see the bullet point when I view in High contrast mode on PC with a black background
- Should look ok with no CSS e.g. I can see a 'bullet' for each list item
- View the feature with text resized to 200% - the 'bullet' should be proportionally larger
- View the feature with the page zoomed to 200%- the 'bullet' should be zoomed proportionally

**Comment by rebeccamcginn** [comment]
![image](https://user-images.githubusercontent.com/59573468/93870712-864a6200-fcc5-11ea-838a-bfee2398b881.png)

**Comment by rebeccamcginn** [comment]
https://www.bbc.com/afrique/monde-54242688 with Large text 
![image](https://user-images.githubusercontent.com/59573468/93870884-c1e52c00-fcc5-11ea-853e-c87deaacfb93.png)

**Comment by RichardPK** [comment]
Firefox is a doughnut-free zone

![Screenshot 2020-09-22 at 11 26 29](https://user-images.githubusercontent.com/9042672/93871388-7b440180-fcc6-11ea-988c-1960a20767d0.png)
